src/clp_ffi_py/modules/clp_four_byte_encoder.cpp

```diff
@@ -0,0 +1,103 @@
+#include <clp_ffi_py/Python.hpp>
+#include <clp_ffi_py/encoder/encoding_methods.hpp>
+
+// NOLINTBEGIN(*-avoid-c-arrays)
+// NOLINTBEGIN(cppcoreguidelines-avoid-non-const-global-variables)
+// NOLINTBEGIN(modernize-use-trailing-return-type)
+// NOLINTBEGIN(readability-identifier-naming)
+
+PyDoc_STRVAR(
+        encode_preamble_doc,
```

Comment:
Technically these are constants, so they should follow the constant-naming convention (`cEncodePremableDoc`)